wdl2cwl/main.py

```diff
@@ -561,6 +570,16 @@ def get_expr_ifthenelse(self, wdl_ifthenelse: WDL.Expr.IfThenElse) -> str:
 
     def get_expr_apply(self, wdl_apply_expr: WDL.Expr.Apply) -> str:
         """Translate WDL Apply Expressions."""
+        binary_ops = {
```

Comment:
Some operations are binary with just a left operand, right operand and an operator. This dictionary stores function names as keys and their operator symbol as values.

Reply:
Great simplification! :+1: